newdle/core/app.py

```diff
@@ -97,4 +97,5 @@ def create_app(config_override=None, use_env_config=True):
     app.add_url_rule('/', 'index', build_only=True)
     app.add_url_rule('/newdle/<code>', 'newdle', build_only=True)
     app.add_url_rule('/newdle/<code>/<participant_code>', 'newdle', build_only=True)
+    app.add_url_rule('/newdle/<newdleCode>/summary', 'newdle', build_only=True)
```

Comment:
`<code>` to be consistent

Reply:
@ThiefMaster if it is changed to code, the url_for will go for ` app.add_url_rule('/newdle/<code>', 'newdle', build_only=True)` instead of ` app.add_url_rule('/newdle/<code>/summary', 'newdle', build_only=True)` 

Am I missing a way to specify the last part of the url in the url_for?

Reply:
oh, you have to use a different endpoint name, e.g. `summary` or `newdle_summary`.

(for the first two using the same one is fine because it's clear which one to build from the params)